java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -108,6 +108,34 @@ public void get_missing_value(BaseClient client) {
         assertNull(data);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void del_multiple_keys(BaseClient client) {
+        String key1 = "{key}" + UUID.randomUUID();
```

Comment:
For consistency should we think about changing all IT test keys to this format "{key}" + UUID.randomUUID() for future PR.